lib/std/start.zig

```diff
@@ -291,6 +297,8 @@ fn _start() callconv(.Naked) noreturn {
             \\ mov fp, #0
             \\ mov lr, #0
             \\ mov x0, sp
+            \\ and x1, x0, #-16
+            \\ mov sp, x1
```

Comment:
```suggestion
            \\ and sp, x0, #-16
```
I know it's annoying to remember which parts of which aarch64 instructions can access sp :roll_eyes: